flutter/lib/src/default_integrations.dart

```diff
@@ -8,6 +8,40 @@ import 'package:sentry/sentry.dart';
 import 'sentry_flutter_options.dart';
 import 'widgets_binding_observer.dart';
 
+class RunZoneGuardedIntegration extends Integration<SentryFlutterOptions> {
+  RunZoneGuardedIntegration(this._runBeforeIntegrations, this._runAfterIntegrations);
+
+  final Function() _runBeforeIntegrations;
+  final Function() _runAfterIntegrations;
```

Comment:
as I see, `runAfter` is always a `AppRunner` instance, should we use the typedef instead?